imports/api/groceries.js

```diff
@@ -34,41 +34,33 @@ Meteor.methods({
 		check(groceryId, String);
 		authCheck(Groceries, this.userId, groceryId);
 
-		Groceries.remove(groceryId);
+		return Groceries.remove(groceryId);
```

Comment:
I added the return cause it might be useful in the UI to be able to see the error (if there is one)